cmd/werf/common/common.go

```diff
@@ -1073,6 +1064,45 @@ func GetNamespace(cmdData *CmdData) string {
 	return *cmdData.Namespace
 }
 
+// FIXME: inconsistent behaviour:
+// "WERF_SET_TEST=test=1val" and "--set "test=2val"" results in "test: 2val", but
+// "WERF_SET_TEST=test=3val" and "--set "test=2val"" results in "test: 3val";
+// "...test=true" and "...test=false" in any order results in "test: true"
+func GetSet(cmdData *CmdData) []string {
+	envSet := predefinedValuesByEnvNamePrefix("WERF_SET_", "WERF_SET_STRING_", "WERF_SET_FILE_")
```

Comment:
Why WERF_SET_ + WERF_SET_STRING_+ WERF_SET_FILE_ here?

The desireable behaviour of `GetSet` is to return only `WERF_SET_` params without `WERF_SET_STRING_` and `WERF_SET_FILE_`.